campgns/origplus/map00014.txt

```diff
@@ -23,8 +23,8 @@ SET_GENERATE_SPEED(500)
 COMPUTER_PLAYER(PLAYER1,0)
 COMPUTER_PLAYER(PLAYER2,0)
 
-SET_COMPUTER_GLOBALS(PLAYER1,60,1,1,1,1900,1)
-SET_COMPUTER_GLOBALS(PLAYER2,70,1,2,1,1700,1)
+SET_COMPUTER_GLOBALS(PLAYER1,60,1,1,1,1900,1,0)
```

Comment:
This is still from the original script, that campaign has it too. I fixed the documentation of SET_COMPUTER_GLOBALS on the wiki. That final param is optional.